packages/components/src/components/ScalarPopover/ScalarPopover.vue

```diff
@@ -23,6 +23,7 @@ const { cx } = useBindCx()
       <template #floating="{ width, height }">
         <PopoverPanel
           v-slot="{ close }"
+          :middleware="middleware"
```

Comment:
I don't think this will do anything here since the middleware is handled by the floating element 🤔